sensor/docs/tutorials/marker_radar_lidar_calibrator.md

```diff
@@ -0,0 +1,153 @@
+# marker_radar_lidar_calibrator
+
+In this tutorial, we will present a hands-on tutorial of the `marker_radar_lidar_calibrator`. Although we provide pre-recorded rosbags, the flow of the tutorial is meant to show the user the steps they must perform in their own use cases with live sensors.
+
+General documentation regarding this calibrator can be found [here](../../marker_radar_lidar_calibrator/README.md).
+
+## Setup
+
+This tutorial assumes that the user has already built the calibration tools.
+Installation instructions can be found [here](../../README.md)
+
+## Data preparation
+
+Please download the data (rosbag) from [here](https://drive.google.com/drive/folders/1S3Cz_VomvHBRgiCSt8JCOgN53UGz5TpZ).
+
+The rosabg includes four different topics including `object_raw`, `pointcloud_raw`, and `/tf_static`.
+
+## Environment preparation
+
+### Overall calibration environment
+
+The required space for calibration depends on the vehicle and sensors used. For a normal consumer-level car, a space of `5m x 10m` should be sufficient.
+
+### Radar reflector
+
+Radar reflector is the only moving element during the calibration process and must be detected by both radar and lidar. It is recommended to utilize a tripod to adjust the height of the radar reflector and also modify its center to align with the radar sensor.
+
+## Launching the tool
+
+In this tutorial, we will use the X2 vehicle of Tier IV.
+First, run the sensor calibration manager:
+
+```bash
+ros2 run sensor_calibration_manager sensor_calibration_manager
+```
+
+In `project`, select `x2`, and in `calibrator`, select `marker_radar_lidar_calibrator`. Then, press `Continue`.
+
+<p align="center">
+    <img src="../images/marker_radar_lidar_calibrator/menu1.jpg" alt="menu1">
+</p>
+
+A menu titled `Launcher configuration` should appear in the UI, and the user may change any parameter he deems convenient.
+For this tutorial, we will modify the default value `radar_name` from `front_left` to `front_center`. For the `msg_type` and `transformation_type`, as `object_raw` topic is type of `radar_tracks` and the radar type is 2D in this tutorial, we keep them as default. After configuring the parameters, click `Launch`.
```

Comment:
If transformation type is the new formulation for the 4D radars, it should not be here. That PR will not be merged in several months (we do not have good data for a proper evaluation)

Reply:
updated the image and remove the content in the newest version.